File: symphony/pidev5/pidev2/src/Freelance/FrontOfficeBundle/Entity/Membre.php
<?php

namespace Freelance\FrontOfficeBundle\Entity;
use FOS\UserBundle\Model\User as BaseUser;
use Doctrine\ORM\Mapping as ORM;

/**
 * Membre
 *
 * @ORM\Table(name="membre", indexes={@ORM\Index(name="pseudo", columns={"pseudo"})})
 * @ORM\Entity
 */
class Membre extends BaseUser
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    protected $id;

    /**
     * @var string
     *
     * @ORM\Column(name="nom", type="string", length=30, nullable=false)
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="prenom", type="string", length=30, nullable=false)
     */
    private $prenom;

    /**
     * @var string
     *
     * @ORM\Column(name="pays", type="string", length=40, nullable=false)
     */
    private $pays;

    /**
     * @var string
     *
     * @ORM\Column(name="ville", type="string", length=50, nullable=false)
     */
    private $ville;

    /**
     * @var string
     *
     * @ORM\Column(name="pseudo", type="string", length=50, nullable=false)
     */
    private $pseudo;




    /**
     * @var string
     *
     * @ORM\Column(name="photo", type="blob", nullable=true)
     */
    private $photo;

 private $upload;
 public function getUpload() {
     return $this->upload;
 }

 public function setUpload($upload) {
     $this->upload = $upload;
 }

     /**
     * Get id
     *
     * @return integer 
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set nom
     *
     * @param string $nom
     * @return Membre
     */
    public function setNom($nom)
    {
        $this->nom = $nom;

        return $this;
    }

    /**
     * Get nom
     *
     * @return string 
     */
    public function getNom()
    {
        return $this->nom;
    }

    /**
     * Set prenom
     *
     * @param string $prenom
     * @return Membre
     */
    public function setPrenom($prenom)
    {
        $this->prenom = $prenom;

        return $this;
    }

    /**
     * Get prenom
     *
     * @return string 
     */
    public function getPrenom()
    {
        return $this->prenom;
    }

    /**
     * Set pays
     *
     * @param string $pays
     * @return Membre
     */
    public function setPays($pays)
    {
        $this->pays = $pays;

        return $this;
    }

    /**
     * Get pays
     *
     * @return string 
     */
    public function getPays()
    {
        return $this->pays;
    }

    /**
     * Set ville
     *
     * @param string $ville
     * @return Membre
     */
    public function setVille($ville)
    {
        $this->ville = $ville;

        return $this;
    }

    /**
     * Get ville
     *
     * @return string 
     */
    public function getVille()
    {
        return $this->ville;
    }

    /**
     * Set pseudo
     *
     * @param string $pseudo
     * @return Membre
     */
    public function setPseudo($pseudo)
    {
        $this->pseudo = $pseudo;

        return $this;
    }

    /**
     * Get pseudo
     *
     * @return string 
     */
    public function getPseudo()
    {
        return $this->pseudo;
    }

   

    

    /**
     * Set photo
     *
     * @param string $photo
     * @return Membre
     */
    public function setPhoto($photo)
    {
        $this->photo = $photo;

        return $this;
    }

    /**
     * Get photo
     *
     * @return string 
     */
    public function getPhoto()
    {
        return $this->photo;
    }
    public function __toString() {
        return $this->pseudo;
    }

    
}

File: j2me final/pidev_mobile_java/pidevMobile/src/pi/Membre.java
package pi;

public class Membre {

    private int id;
    private String nom;
    private String prenom;
     private String pays;
    private String ville;
    private String pseudo;
    private String password;
    private String email;
    private String societe;
    private String disponibilite;

    public Membre(int id, String prenom, String pays, String ville, String pseudo, String password, String email) {
        this.id = id;
        this.prenom = prenom;
        this.pays = pays;
        this.ville = ville;
        this.pseudo = pseudo;
        this.password = password;
        this.email = email;
        
    }

    public Membre(int id, String nom, String prenom, String pays, String ville, String pseudo, String email, String societe) {
        this.id = id;
        this.nom = nom;
        this.prenom = prenom;
        this.pays = pays;
        this.ville = ville;
        this.pseudo = pseudo;
        this.email = email;
        this.societe = societe;
    }

    public Membre(int id, String nom, String prenom, String pays, String ville, String pseudo, String password, String email, String disponibilite) {
        this.id = id;
        this.nom = nom;
        this.prenom = prenom;
        this.pays = pays;
        this.ville = ville;
        this.pseudo = pseudo;
        this.password = password;
        this.email = email;
        this.disponibilite = disponibilite;
    }

    public String getSociete() {
        return societe;
    }

    public void setSociete(String societe) {
        this.societe = societe;
    }

    public String getDisponibilite() {
        return disponibilite;
    }

    public void setDisponibilite(String disponibilite) {
        this.disponibilite = disponibilite;
    }
    
    
    


    public Membre() {
    }

    public int getId() {
        return id;
    }

    public String getNom() {
        return nom;
    }

    public String getPrenom() {
        return prenom;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public void setPrenom(String prenom) {
        this.prenom = prenom;
    }

    void setId(String id) {
        this.id = Integer.parseInt(id);
    }

    public String getPays() {
        return pays;
    }

    public void setPays(String pays) {
        this.pays = pays;
    }

    public String getVille() {
        return ville;
    }

    public void setVille(String ville) {
        this.ville = ville;
    }

    public String getPseudo() {
        return pseudo;
    }

    public void setPseudo(String pseudo) {
        this.pseudo = pseudo;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String toString() {
        return "Membre{" + "id=" + id + ", nom=" + nom + ", prenom=" + prenom +", pays=" + pays + 
                ", ville=" + ville +", pseydo=" + pseudo +", password=" + password +", email=" + email +'}';
    }
    public String toStringe() {
        return "Membre{" + "id=" + id + ", nom=" + nom + ", prenom=" + prenom +", pays=" + pays + 
                ", ville=" + ville +", pseydo=" + pseudo +", password=" + password +", email=" + email +", societe=" + societe+ '}';
    }
    public String toStringee() {
        return "Membre{" + "id=" + id + ", nom=" + nom + ", prenom=" + prenom +", pays=" + pays + 
                ", ville=" + ville +", pseydo=" + pseudo +", password=" + password +", email=" + email +", disponibilite=" + disponibilite +'}';
    }

    Membre getMembre() {
        return null;
    }
}
